aquadoggo/src/domain.rs

```diff
@@ -503,7 +503,7 @@ mod tests {
     #[rstest]
     #[case::ok(&[(0, 8)], (0, 8))]
     #[should_panic(
-        expected = "Expected skiplink target for <Author 53fc96> at log id 0 and seq num 4 not found in database"
+        expected = "Expected skiplink target not found in store: <Author 53fc96>, log id 0, seq num 4"
```

Comment:
I thought it might be handy to structure the log so that it names the issue first (not found) and then gives the details (ids, etc.), then it's easier to read.